part2/stages/Functions/install-main

```diff
@@ -855,10 +855,15 @@ install_file()
 
     local SRC="$1"
     local DST="$2"
+    local IMG_TYPE="$3"
 
```

Comment:
Nitpicking but this should probably be named PACKAGE_TYPE instead of IMG_TYPE.

Reply:
Sure, I can change that.  Thanks.